### DISCUSSION TITLE
Dont work

### DISCUSSION THREAD

**Discussion by mgoldens** [nielsmaerten/gluco-check#159] [Q&A] [answered]
I can't link the google assistant with gluco check. When I invoke it from a routine it brings me results that are not correct. Tried various options and can't get it to work. Any help you can give me? (The assistant is in Spanish but I already tried putting the phrases in English and it doesn't work either). Thanks!

**Comment by nielsmaerten** [comment, marked as answer]
@mgoldens 

If you want to to invoke Gluco Check in English, you must have English enabled as one of the languages for Google Assistant.
Do your Assistant Language Settings look like this?:
![image](https://user-images.githubusercontent.com/4604406/138699697-cedffe6c-9ada-4b66-a65b-78b39571ab3e.png)

From the Google-app, you can get to this screen by tapping Settings => Assistant => Languages. Or by saying "ok google, show assistant language settings". (or its equivalent in Spanish)

Note: You can't use Gluco Check in Spanish yet, as the translation is not yet complete. But if you want, I can add you as a beta tester so you can try out the Spanish version.

**Comment by mgoldens** [comment]
Hello! I tried what you tell me and although it didn't work yesterday, today it works! I hope it continues like this !! I would love to be part of the beta in Spanish. How should I do to be included? Thanks!!

**Reply by nielsmaerten** [reply]
Send an email to beta@glucocheck.app, and I'll let you know as soon as we start testing the Spanish version :)
Make sure to send the email form same address as the one you're using for your Google Account. (that's likely an @gmail.com address)

**Reply by mgoldens** [reply]
Niels

I already sent the mail as you indicated.

Thanks a lot.

Marcos
